weapons/defaultweaponclass.yaml

```diff
@@ -129,7 +129,7 @@ BlueFireballLauncher:
 		Shadow: true
 		Inaccuracy: 0c181
 		Image: missile01
-		HorizontalRateOfTurn: 8
+		VerticalRateOfTurn: 12
```

Comment:
oh, good that you've noticed, theoretically the default value is 5, but I better return it